### DISCUSSION TITLE
flashbang is useless in current state

### DISCUSSION THREAD

**Discussion by 0rangeGhost** [berkayylmao/NFS-Chat-Chaos-Mod#19] [General]
the game is pretty bright itself, so it doesn't blinds you with excessive brightness, and it doesn't obscures vision cos it vanishes pretty fast. i guess it could be prolonged to be noticeable

**Comment by berkayylmao** [comment]
There really isn't more that effect can do. The flashbang effect in almost all games is just a white screen and some random visual distortion for 3-5 seconds. Turning issue into a discussion to see if someone can come up with a viable improvement.
